workflows/cloud-client/execute_with_arguments_test.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2025 Google LLC
```

Comment:
This is a moved file, not a new file, the copyright header doesn't need to be updated.